Add enhancements to poker hand tips

nh6574 opened the pull request:
Added as a third value to the examples

![Captura de pantalla 2025-05-04 094403](https://github.com/user-attachments/assets/73ff5a31-4d76-4914-8558-a9a87dbc6605)
![2379780_20250504094433_1](https://github.com/user-attachments/assets/e227756b-ed7f-4516-b858-59640c67cd4d)

## Additional Info:
<!-- Don't worry too much if you don't know what these are or how to fill them. It's mostly reminders for maintainers ;) -->
- [ ] I didn't modify api's or I've made a PR to the [wiki repo](https://github.com/Steamodded/wiki).
- [ ] I didn't modify api's or I've updated lsp definitions.
- [ ] I didn't make new lovely files or all new lovely files have appropriate priority.

english5040 commented:
On first glance I don't like the implementation- it's not general enough and we love generality in APIs.
The general idea is good

BakersDozenBagels commented:
You could always require something like `{ 'H_K', true, enhancement = 'm_gold' }`. With that, adding keys for other modifications isn't horrifically clunky.

nh6574 commented:
Makes sense to me. I can add that if there's no second opinion

nh6574 commented:
Changed it to @BakersDozenBagels's suggestion and added editions and seals as well. Could probably be expanded to stickers too.

![image](https://github.com/user-attachments/assets/e232d465-6242-4740-a25d-142f23866b8a)
![Captura de pantalla 2025-05-06 211815](https://github.com/user-attachments/assets/faa595c0-911a-450c-b525-2d4d0992180f)

english5040 commented:
Alright, this looks good
